classes/db/DbPDO.php

```diff
@@ -126,6 +126,11 @@ public function connect()
         } catch (PDOException $e) {
             throw new PrestaShopException('Link to database cannot be established: ' . $e->getMessage());
         }
+        
+        // UTF-8 support
+        if ($this->link->exec('SET NAMES \'utf8\'') === false) {
+			throw new PrestaShopException('PrestaShop Fatal error: no utf-8 support. Please check your server configuration.'.$e->getMessage());
+        }
```

Comment:
Isn't this SQL request a duplicate of `PDO::MYSQL_ATTR_INIT_COMMAND => "SET NAMES utf8"`?